### ISSUE TITLE
FashionNet: Landmark detection

### ISSUE THREAD

**Issue by libary753** [libary753/SKKU_KYS#4]
VGG16을 약간 변형하여 옷에서 Landmark를 찾아내는 인공신경망 만들기

**Comment by libary753** [comment]
현재 VGG16을 변경한 망까지는 구축 완료, Loss Function은 논문에 나와있는 것으로 사용하여 학습 진행해야함

**Comment by libary753** [comment]
visibility, clothes type, variation type : cross entropy loss function
landmark, bound box : linear distance 사용
bias: 0
std: 0.03 사용
학습실패

**Comment by libary753** [comment]
<cross entropy loss function 에 대해서만 학습>
clothes type, variation type : cross entropy loss function
bias: 0
std: 0.03 사용
learning rate: 0.0001
학습실패

**Comment by libary753** [comment]
Full model 학습 완료.
정확도가 100%는 아니지만 어느 정도 수준 이상은 보임.
0~3 epoch : L.R: 0.0001 Loss: Loss_visibility+Loss_position
4~10 epoch : L.R: 0.00001 Loss: 20*Loss_visibility+Loss_position
11~15 epoch: L.R: 0.000001 Loss: 20*Loss_visibility+Loss_position
